From 1fd737d1dc6d867ae899326c3e18ae230f12f3d2 Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Thu, 19 Sep 2019 23:45:46 -0700
Subject: [PATCH 0670/1851] dt-bindings: display: xlnx: dsi: Add command mode
 support

commit da387e7475bef361d5ca4fb60fb96caf60563fe6 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds the command mode support as optional entry.

Signed-off-by: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt
index 55508167e606..a545a0d818e6 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt
@@ -46,6 +46,7 @@ Required simple_panel properties:
 Optional properties:
  - xlnx,vpss: vpss phandle
    This handle is required only when VPSS is connected to DSI as bridge.
+ - xlnx,dsi-cmd-mode: denotes command mode enable.
 
 Example:
 
-- 
2.31.1

